raven/src/main/java/com/getsentry/raven/connection/AsyncConnection.java

```diff
@@ -84,7 +81,7 @@ public AsyncConnection(Connection actualConnection, ExecutorService executorServ
      * @param gracefulShutdown Indicates whether or not the shutdown operation should be managed by a ShutdownHook.
      */
     public AsyncConnection(Connection actualConnection, ExecutorService executorService, boolean gracefulShutdown) {
-        this(actualConnection, executorService, gracefulShutdown, DEFAULT_SHUTDOWN_TIMEOUT);
+        this(actualConnection, executorService, gracefulShutdown, DefaultRavenFactory.ASYNC_SHUTDOWN_TIMEOUT_DEFAULT);
```

Comment:
I'm OK with 2, but wasn't sure about dropping them without deprecation. Probably not a big deal?

The reason 1 stinks is it means my `getSomething` method has no default, and so now must return `null` or some sentinel value which made me sad considering every other new getter was always expected to return a "real" value. Felt gross to me to have some that could return `null` and some that couldn't.